src/components/main/Main.tsx

```diff
@@ -35,8 +36,14 @@ const StyledLink = styled(Link)(() => ({
   display: 'flex',
   alignItems: 'center',
 }));
-const LinkComponent = ({ children }: { children: JSX.Element }) => (
-  <StyledLink to={HOME_PATH}>{children}</StyledLink>
+const LinkComponent = ({ children }: { children: ReactNode }) => (
+  <StyledLink
+    data-umami-event="header-home-link"
```

Comment:
I wonder if it's okay to send `header.home-link`? so the scope is clearer?

Reply:
I think the event names should be consistent, not sure we need to add complexity to them. We can discuss the naming scheme